src/screens/novel/components/EditInfoModal.tsx

```diff
@@ -38,7 +38,13 @@ const EditInfoModal = ({
     setNovel({ ...novel, genres: tags?.join(',') });
   };
 
-  const status = ['Ongoing', 'Hiatus', 'Completed', 'Unknown', 'Cancelled'];
+  const status = [
+    getString('novelScreen.status.onGoing'),
+    getString('novelScreen.status.hiatus'),
+    getString('novelScreen.status.completed'),
+    getString('novelScreen.status.unknown'),
+    getString('novelScreen.status.cancelled'),
+  ];
```

Comment:
This will make the status icon in novel info is always `?` if non-english users change the status

Reply:
How can we translate status? use Enums with display function which calls getString?